worlds/dark_souls_3/__init__.py

```diff
@@ -1,21 +1,14 @@
 # world/dark_souls_3/__init__.py
-from typing import Dict
-
-from .Items import DarkSouls3Item
-from .Locations import DarkSouls3Location
-from .Options import dark_souls_options
-from .data.items_data import weapons_upgrade_5_table, weapons_upgrade_10_table, item_dictionary, key_items_list, \
-    dlc_weapons_upgrade_5_table, dlc_weapons_upgrade_10_table
-from .data.locations_data import location_dictionary, fire_link_shrine_table, \
-    high_wall_of_lothric, \
-    undead_settlement_table, road_of_sacrifice_table, consumed_king_garden_table, cathedral_of_the_deep_table, \
-    farron_keep_table, catacombs_of_carthus_table, smouldering_lake_table, irithyll_of_the_boreal_valley_table, \
-    irithyll_dungeon_table, profaned_capital_table, anor_londo_table, lothric_castle_table, grand_archives_table, \
-    untended_graves_table, archdragon_peak_table, firelink_shrine_bell_tower_table, progressive_locations, \
-    progressive_locations_2, progressive_locations_3, painted_world_table, dreg_heap_table, ringed_city_table, dlc_progressive_locations
-from ..AutoWorld import World, WebWorld
+from typing import Dict, Set, List
+
 from BaseClasses import MultiWorld, Region, Item, Entrance, Tutorial, ItemClassification
-from ..generic.Rules import set_rule, add_item_rule
+from Options import Toggle
+
+from .Items import DarkSouls3Item, DS3ItemCategory, item_dictionary, key_item_names
+from .Locations import DarkSouls3Location, DS3LocationCategory, location_tables, location_dictionary
+from .Options import RandomizeWeaponLevelOption, PoolTypeOption, dark_souls_options
+from ..AutoWorld import World, WebWorld
+from ..generic.Rules import set_rule, add_rule, add_item_rule
```

Comment:
```suggestion
from worlds.AutoWorld import World, WebWorld
from worlds.generic.Rules import set_rule, add_rule, add_item_rule
```
see https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/apworld%20specification.md#caveats

Reply:
Changed in 44a3751

This was the only thing required to swap DS3 to be an apworld, so I've also removed it from the list of shame.

Tested by running `setup.py build`, replacing my ds3 folder in my 0.4.1 install with the built apworld, and generating a couple seeds.